boards/st/nucleo_h533re/support/openocd.cfg

```diff
@@ -0,0 +1,25 @@
+#openOCD from https://github.com/STMicroelectronics/OpenOCD
```

Comment:
Should we use ST-Openocd or main community openocd (https://github.com/openocd-org/openocd) ?